panels/model_checks.py

```diff
@@ -0,0 +1,44 @@
+from panels import *
```

Comment:
The ``Event`` stuff is moved here from the core repo.  The rest is new, and hopefully seems straightforward.